piet-direct2d/Cargo.toml

```diff
@@ -14,12 +14,9 @@ piet = { version = "0.2.0", path = "../piet" }
 utf16_lit = "1.0"
 
 wio = "0.2.2"
-
-[dependencies.winapi]
-version = "0.3.8"
-features = ["d2d1", "d2d1_1", "d2d1effects", "d3d11", "dxgi", "winnls"]
+winapi = { version = "0.3.8", features = ["d2d1", "d2d1_1", "d2d1effects", "d3d11", "dxgi", "winnls"] }
```

Comment:
Time to upgrade to 0.3.9 while we're doing this?

Reply:
will do that as a separate patch.